trusted-types/trusted-types-eval-reporting.html

```diff
@@ -39,7 +39,8 @@
   // Like assert_throws_*, but we don't care about the exact error. We just want
   // to run the code and continue.
   function expect_throws(fn) {
-    try { fn(); assert_unreached(); } catch (err) { /* ignore */ }
+    try { fn(); } catch (err) { return; /* ignore */ }
```

Comment:
Why is this better?

Reply:
Current behavior is broken, see commit message.

Reply:
Ah apologies yeah that makes sense.